feature-detects/es6/string-template.js

```diff
@@ -0,0 +1,26 @@
+/*!
+{
+  "name": "ES6 Template Strings",
+  "property": "stringtemplate",
+  "caniuse": "template-literals",
+  "notes": [{
+    "name": "ECMAScript 6 draft specification",
+    "href": "https://tc39wiki.calculist.org/es6/template-strings/"
+  }],
+  "authors": ["dabretin"],
+  "tags": ["es6"]
+}
+!*/
+/* DOC
+Check if browser implements ECMAScript 6 String template.
+*/
+define(['Modernizr'], function(Modernizr) {
+  Modernizr.addTest('stringtemplate', function() {
+    try {
+      // eslint-disable-next-line
+      return eval('(function(){var a=1; return `-${a}-`;})()') == '-1-';
```

Comment:
shouldnt this be a strict comparison `===` ?

Reply:
I don't think it's necessary ('-1-' can only be a string) but your're right: never too many precautions !